src/batch-submitter/batch-submitter.ts

```diff
@@ -30,6 +30,12 @@ export interface Range {
   start: number
   end: number
 }
+export interface ResubmissionConfig {
+  resubmissionTimeout: number,
+  minGasPrice: number,
+  maxGasPrice: number,
```

Comment:
oo nice to double check this, [ynatm](https://github.com/kendricktan/ynatm) passes in a `.toGwei()`ed value, so that means we pass in a gwei unit right? is there a unit we consistently use in the rest of the code base @karlfloersch? still learning standard practice around gas 😅

Reply:
The `gasScalingFunction.LINEAR` is used, which takes a second argument `inGwei` which defaults to `true`. Internally `minGasPrice` and `maxGasPrice` are passed into `gasScalingFunction.LINEAR`, so yes they are denominated in gwei. We could denominate them in `wei` if we pass the second arg `false` to `LINEAR`
See: https://github.com/kendricktan/ynatm/blob/ddeb20e39d8f4bad8c9a1890a1a3af30838bcee9/index.js#L16

We do use ether already in the env vars, I think consistency is important - `SAFE_MINIMUM_ETHER_BALANCE`. The env vars should either have `GWEI` in their name or be denominated in ether, but I think from a UX perspective, gwei is more commonly used when thinking about gas prices so we should just add `GWEI` to the env var names

Reply:
updated